bccsp/handlers/nym.go

```diff
@@ -160,7 +160,10 @@ func (i *NymPublicKeyImporter) KeyImport(raw interface{}, opts bccsp.KeyImportOp
 
 	pk, err := i.User.NewPublicNymFromBytes(bytes)
 	if err != nil {
-		return nil, err
+		pk, err = i.User.NewPublicNymFromBytes(append([]byte{04}, bytes...))
```

Comment:
can you add a comment to clarify why appending that magic number?
Can you put it in a const?

Reply:
Good point. So, there are several issues here at play. I'll try to summarise them here and maybe you can give me your take on how to best document it? So:

1. the `NymPublicKeyImporter` does not have a `translator` to handle the various byte representations of group elements. Instead, it directly passes the bytes to the underlying implementation without any translation. This works well most of the times since actually for most curves, the serialisation of a group element is a fixed-size array of the two coordinates. However some curves use an encoding where the constant byte value `0x04` is prefixed to the two coordinates to signal that the point is uncompressed.
2. it is meaningful to add the translator, but I would do it in a separate change
3. prefixing `0x04` cannot turn a bad point marshalling into a good one anyway, unless it's the encoding issue that we are handling.

Wdyt?

Reply:
I would add in the comments points 1 and 2, and then open a github issue for the translator 👍

Reply:
Done